packages/frontend/app/components/app/sidebar/chats/channel-form/index.ts

```diff
@@ -6,7 +6,7 @@ import { action } from '@ember/object';
 
 import ChannelManager from 'emberclear/services/channel-manager';
 import { dropTask } from 'ember-concurrency-decorators';
-import Task from 'ember-concurrency/task';
+import { taskFor } from 'emberclear/utils/ember-concurrency';
```

Comment:
these changes shouldn't be a part of your diff, as they were changes I had made

Reply:
try the following: 
```
git fetch upstream
git checkout master
git reset upstream/master --hard
git checkout feature-vote-verification
git rebase master
git push origin feature-vote-verification --force-with-lease
```

Reply:
It looks like my most recent rebase went awry

Reply:
I rebased my master branch with upstream master then rebased my feature branch with local master. Would that cause these issues?

Reply:
That works.

Reply:
forked masters often get out of sync.
I never trust my own local copy of master and always reset from upstream when I rebase and create new branches